src/module/Ui/config/module.config.php

```diff
@@ -222,6 +223,11 @@
                 ]
             ]
         ],
+        'rendererToStrategy' => [
```

Comment:
Still use Assetic to bundle the old assets, but don't include them anymore (since this was made explicit)